### PR TITLE
Configuration support

### PR DESCRIPTION
**Pull request by Ivy-Wusky**

### CODE REVIEW

**Review by Grass0001 — CHANGES_REQUESTED** [review]
image
i must request a note with "sex" somewhere, and "youarenotgettingmybottoken" i believe is a fundamental part of the program.

**Comment by LoserIsPoggers** [comment]
I love ivy bot

.yiff when

File: README.md [hunk 1]
```diff
@@ -5,33 +5,26 @@ click [here](https://discord.com/api/oauth2/authorize?client_id=7199335798654894
...
+    <add key="Password" value="replace with the set password"/>
+  </appSettings>  
+</configuration>  
 ```
 
-make sure to pass the **same** configuration in both and if you are running your **lavalink** instance on [heroku](https://www.heroku.com) the port will always be **80** regardless of the port you set
+remember that if you are running your **lavalink** instance on [heroku](https://www.heroku.com) the port will always be **80** regardless of the port you set
 # future plans
-customizable configuration most likely in the **json** or **xml** format which would mean not having to edit source files at all
+additional commands and an **nsfw** module
```
Comment (Grass0001) [inline README.md:27]: 😳

**Comment by Ivy-Wusky** [comment]
might add prefix as well

**Comment by Ivy-Wusky** [comment]
ready to merge

**Pull request merged**